proxy/mitmproxy.py

```diff
@@ -61,7 +61,7 @@ def command(self):
                 "--wfile",
                 self.path,
                 "--script",
-                " ".join([os.path.join(self.scripts, "inject-deterministic.py")]),
+                " ".join([os.path.join(self.scripts, "record_scripts.py")]),
```

Comment:
This no longer exists, does mitm 2 support multiple scripts specified on the command line?

Reply:
Updated to make all work on mitm2.0.2 also